org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/editor/copyqualifiedname/XbaseCopyQualifiedNameService.java

```diff
@@ -0,0 +1,197 @@
+		return toQualifiedName(jvmExecutable);
+	}
+
+	protected String _getQualifiedName(JvmExecutable executable, XAbstractFeatureCall featureCall) {
+		IResolvedTypes resolvedTypes = typeResolver.resolveTypes(featureCall);
+		IResolvedExecutable resolvedExecutable = resolveExecutable(executable, featureCall, resolvedTypes);
+		return toFullyQualifiedName(executable) //
+				+ "(" + toQualifiedNames(featureCall.getActualArguments(), //
+						(argument) -> toQualifiedName(argument, resolvedExecutable, executable, resolvedTypes,
+								featureCall.getActualArguments()))
+				+ ")";
+	}
+
+	protected String _getQualifiedName(JvmConstructor constructor, XConstructorCall constructorCall) {
+		IResolvedTypes resolvedTypes = typeResolver.resolveTypes(constructorCall);
+		IResolvedExecutable resolvedExecutable = resolveExecutable(constructor, constructorCall, resolvedTypes);
+		return toFullyQualifiedName(constructor) //
+				+ "(" + toQualifiedNames(constructorCall.getArguments(), //
+						(argument) -> toQualifiedName(argument, resolvedExecutable, constructor, resolvedTypes,
+								constructorCall.getArguments()))
+				+ ")";
+	}
+
+	protected IResolvedExecutable resolveExecutable(JvmExecutable consturctor, XExpression expression, IResolvedTypes resolvedTypes) {
+		if (consturctor instanceof JvmConstructor && expression instanceof XAbstractFeatureCall) {
+			return _resolveExecutable((JvmConstructor) consturctor, (XAbstractFeatureCall) expression, resolvedTypes);
+		} else if (consturctor instanceof JvmConstructor && expression instanceof XConstructorCall) {
+			return _resolveExecutable((JvmConstructor) consturctor, (XConstructorCall) expression, resolvedTypes);
+		} else if (consturctor instanceof JvmOperation && expression instanceof XAbstractFeatureCall) {
+			return _resolveExecutable((JvmOperation) consturctor, (XAbstractFeatureCall) expression, resolvedTypes);
+		} else {
+			throw new IllegalArgumentException(
+					"Unhandled parameter types: " + Arrays.asList(consturctor, expression, resolvedTypes).toString());
+		}
+	}
+
+	protected IResolvedExecutable _resolveExecutable(JvmConstructor consturctor, XAbstractFeatureCall featureCall,
+			IResolvedTypes resolvedTypes) {
+		XExpression actualReceiver = featureCall.getActualReceiver();
+		LightweightTypeReference actualType = null;
+		if (actualReceiver != null) {
+			actualType = resolvedTypes.getActualType(actualReceiver);
+		}
+
+		ResolvedFeatures resolvedFeatures = null;
+		if (actualType != null) {
+			resolvedFeatures = overrideHelper.getResolvedFeatures(actualType);
+		}
+
+		List<IResolvedConstructor> declaredConstructors = null;
+		if (resolvedFeatures != null) {
+			declaredConstructors = resolvedFeatures.getDeclaredConstructors();
+		}
+
+		Optional<IResolvedConstructor> resolvedConstructor = FluentIterable.from(declaredConstructors)
+				.filter(constr -> constr.getDeclaration().equals(consturctor)) //
```

Comment:
constructor*

Reply:
Let me change it to `declaredConstructor` to be consistent with the list `declaredConstructors`.